schema/index/annotation.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2019-present Facebook Inc. All rights reserved.
```

Comment:
@sywesk, as mentioned in previous review. Please remove this file from this PR, and we'll add this logic in next one inside the entsql package.

Reply:
For those reading this: 
- schema/*/annotation is the place for generic annotations
- entsql is for the database specific annotations (and this is one)